data/blog/2024-year-in-review.mdx

```diff
@@ -0,0 +1,545 @@
+infancy. The [General Fund](/funds/general) is dedicated towards all other free
+and open-source projects and contributors, especially bitcoin-related projects
+and projects that help bitcoin flourish.
+
+As of December 2024, we have allocated over \$12M in grants from the General
+Fund, and close to \$9M in grants from The Nostr Fund. In terms of money
+allocated [year-over-year](/blog/2023-year-in-review#allocation) this is an
+increase of 67% on the nostr side, and 81% on the bitcoin side. When looking at
+total allocation in USD terms, our two funds are at ~89% and ~52% allocated,
+respectively.
+
+![](/static/images/blog/2024-eoy/allocation.png)
+
+As mentioned in [last year’s report](/blog/2023-year-in-review), we suspect that
+the distinction between “bitcoin” and “nostr” will get more blurry over time, as
+nostr matures and gets integrated into a variety of bitcoin software across the
+stack. This is already happening, especially on the eCash front. Are [Cashu
+addresses](/blog/nostr-grants-july-2024#npubcash) a Lightning thing, a bitcoin
+thing, or a nostr thing? Are wallets that use nostr for social payments nostr
+clients, or bitcoin wallets? And what about P2P exchanges that use nostr to
+publish their order books? It is obvious to us that various concepts and
+technologies are currently merging, and we expect this cross-pollination of
+ideas to continue.
+
+With all that said, nostr is still an incredibly young network. It is just
+coming out of its infancy, which is why we will continue to fund experimental
+and exploratory projects from our nostr fund. Before we dive into the nostr fund
+in more detail, let’s have a look at our General Fund and what projects we
+funded in 2024.
+
+### Bitcoin: General Fund
+
+Out of the ~\$23M donated towards our General Fund during the lifetime of the
+fund, we have allocated ~\$12M towards free and open-source projects so far. In
+terms of gross amounts the top five categories that we gave grants to are
+Bitcoin Core, Lightning, eCash, Education, and Privacy.
+
+![](/static/images/blog/2024-eoy/bitcoin-distribution.png)
+
+Our support for Bitcoin Core is mostly constituted of long-term support grants
+for core devs, namely:
+[Marco](/blog/announcing-lts-grant-program-to-support-bitcoin-core-contributors),
+[Josi](/blog/josi-baker-receives-opensats-lts-grant),
+[Sjors](/blog/sjors-provoost-receives-opensats-lts-grant),
+[Vasil](/blog/vasil-dimov-receives-lts-grant),
+[Gleb](/blog/gleb-naumenko-receives-lts-grant),
+[Furszy](/blog/furszy-receives-lts-grant-for-bitcoin-core),
+[Will](/blog/will-clark-receives-lts-grant-for-bitcoin-core),
+[0xB10C](/blog/0xB10C-receives-lts-grant),
+[Bruno](/blog/bruno-garcia-receives-lts-grant), and
+[Jon](/blog/jon-atack-receives-lts-grant). The last three were added to our LTS
+roster in 2024. In addition to these long-term grants we gave smaller grants to
+[three up-and-coming core developers](/blog/caring-for-bitcoin-core).
+
```

Comment:
Little bit repetitive to discuss LTS twice in one blog post. Could probably merge the earlier section on LTS with this section here.